packages/frontend/themes/dark.scss

```diff
@@ -6,6 +6,7 @@
   $bgImage: url('./images/background_dark.svg'),
   $bgMessageBubbleIncoming: #303030,
   $bgMessageBubbleOutgoing: #354535,
+  $messageOutgoingStatusColor: #cdb7b5,
```

Comment:
Still seems too dark to me, maybe doesn't even pass accessibility contrast test?

Reply:
Well I "borroughed" the color from Android.

I set it to #e4d4d3 which succeeds in the contrast test but is very close to the normal text color.